wemake_python_styleguide/logic/naming/blacklists.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+from typing import Set
+
+from wemake_python_styleguide.constants import VARIABLE_NAMES_BLACKLIST
+from wemake_python_styleguide.types import ConfigurationOptions
+
+
+def variable_names_blacklist_from(options: ConfigurationOptions) -> Set[str]:
```

Comment:
Can we return `frozenset`?

Reply:
And can we use [`lru_cache`](https://docs.python.org/3/library/functools.html#functools.lru_cache) here?

Reply:
> And can we use [`lru_cache`](https://docs.python.org/3/library/functools.html#functools.lru_cache) here?

I don't know the whole architecture and code flow well, but caching seems to be not needed, because this functions should be ran only once (in the `_NameValidator.__init__()` which is created only once in `WringNameVisitor.__init__()` that again should be created only once by checker), am I right?

Reply:
please, let me know if you still think `lru_cache` is needed

Reply:
@sobolevn during working on #1019 I have found that I was wrong about caching and it's needed, because `Checker` is created for each file being checked, sorry!
Should I review other similar cases and add some issue? Or maybe i should just correct `variable_names_blacklist_from()` in new PR?

Reply:
Yes, please. We can create a separate issue and discuss it there.